hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/codec/OMDBDefinition.java

```diff
@@ -231,22 +231,22 @@ String.class, new StringCodec(), OmKeyInfo.class,
           new OmDBSnapshotInfoCodec());
 
   /**
-   * SnapshotRenamedKeyTable that complements the keyTable (or fileTable)
-   * entries of the immediately previous snapshot in the same snapshot
-   * scope (bucket or volume).
+   * SnapshotRenamedTable that complements the keyTable (or fileTable)
+   * and dirTable entries of the immediately previous snapshot in the
+   * same snapshot scope (bucket or volume).
    * <p>
-   * Key renames between the two subsequent snapshots are captured, this
+   * Key/Dir renames between the two subsequent snapshots are captured, this
    * information is used in {@link SnapshotDeletingService} to check if the
-   * renamedKey is present in the previous snapshot's keyTable
+   * renamedKey or renamedDir is present in the previous snapshot's keyTable
    * (or fileTable).
    */
   public static final DBColumnFamilyDefinition<String, String>
       SNAPSHOT_RENAMED_KEY_TABLE =
```

Comment:
Rename this var as well?
```suggestion
  public static final DBColumnFamilyDefinition<String, String>
      SNAPSHOT_RENAMED_TABLE =
```